src/theme/variables/edgeDark.ts

```diff
@@ -121,10 +121,16 @@ export const edgeDark: Theme = {
   loadingIcon: palette.edgeMint,
 
   // Background
-  // backgroundGradientColors: [palette.navyAqua, palette.navyAquaDarker], // For vertical gradient
-  backgroundGradientColors: [palette.darkestNavy, palette.darkAqua],
-  backgroundGradientStart: { x: 0, y: 0 },
-  backgroundGradientEnd: { x: 1, y: 0 },
+  background: {
+    blurR: 100,
+    color: 'black',
+    dotOpacity: 0.4,
+    dots: [
+      { color: '#997833', cx: '45%', cy: '-20%', r: 100 }, // Mid-right
+      { color: '#999999', cx: '-40%', cy: '-40%', r: 90 }, // Top-left
+      { color: '#15cb7f', cx: '-65%', cy: '50%', r: 250 } // Bottom left
```

Comment:
Can we express the dot's radius in percentages?

Reply:
I did it using the `scale` function.